cloud/openstack/os_user.py

```diff
@@ -154,7 +154,8 @@ def main():
         module.fail_json(msg='shade is required for this module')
 
     name = module.params['name']
-    password = module.params['password']
+    # pop the password so as not to impact the shade constructor
+    password = module.pop('password')
```

Comment:
this line will fail with:
password = module.pop('password')
AttributeError: 'AnsibleModule' object has no attribute 'pop'

it should be:
password = module.params.pop('password')